Decred Wallet/Features/Wallet Utils/StartupPinOrPassword.swift

```diff
@@ -82,14 +82,15 @@ struct StartupPinOrPassword {
             return
         }
 
-        let currentPublicPassphrase = currentPassword ?? self.defaultPublicPassphrase
-        let newPublicPassphrase = newPinOrPassword ?? self.defaultPublicPassphrase
+        let currentPublicPassphrase = currentPassword ?? ""
+        let newPublicPassphrase = newPinOrPassword ?? ""
 
         DispatchQueue.global(qos: .userInitiated).async {
             do {
-                let oldPublicPass = (currentPublicPassphrase as NSString).data(using: String.Encoding.utf8.rawValue)!
-                let newPublicPass = (newPublicPassphrase as NSString).data(using: String.Encoding.utf8.rawValue)!
-                try AppDelegate.walletLoader.wallet?.changePublicPassphrase(oldPublicPass, newPass: newPublicPass)
+                let passphraseType = securityType == SecurityViewController.SECURITY_TYPE_PASSWORD ? DcrlibwalletPassphraseTypePass : DcrlibwalletPassphraseTypePin
+                try AppDelegate.walletLoader.multiWallet.changeStartupPassphrase(currentPublicPassphrase.utf8Bits,
```

Comment:
This functions returns the error "invalid_passphrase" and when you try to open the app it crashes with the same error
<img width="414" alt="Screenshot 2020-01-16 at 12 47 43" src="https://user-images.githubusercontent.com/794430/72522642-7937c200-385e-11ea-9989-0b8292f1bebe.png">